packages/rspack/src/builtin-plugin/index.ts

```diff
@@ -279,6 +279,31 @@ export function deprecated_resolveBuiltins(
 		noEmitAssets = true;
 	}
 
+	if (options.experiments.rspackFuture?.disableTransformByDefault) {
+		(
+			[
+				"react",
+				"pluginImport",
+				"decorator",
+				"presetEnv",
+				"emotion",
+				"relay"
+			] as const
+		).forEach(key => {
+			if (builtins[key]) {
+				deprecatedWarn(
+					`'builtins.${key} = ${JSON.stringify(
+						builtins[key]
+					)}' only works for 'rspackFuture.disableTransformByDefault = false', please migrate to ${termlink(
```

Comment:
Let's use the full path.
```suggestion
					)}' only works for 'experiments.rspackFuture.disableTransformByDefault = false', please migrate to ${termlink(
```